### PR TITLE
cut v2.35.0-rc1

### PR DESCRIPTION
**Pull request by Nexucis**
Signed-off-by: Augustin Husson <husson.augustin@gmail.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## 2.35.0-rc0 / 2022-04-14
```
Comment (roidelapluie) [inline CHANGELOG.md:1]:
```suggestion
## 2.35.0-rc1 / 2022-04-14
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+
+* [ENHANCEMENT] Update package `uber.go/auomaxprocs` to support `cgroups2` #10584
+* [BUGFIX] Tracing/GRPC: Set TLS credentials only when insecure is false. #10592
+* [BUGFIX] TSDB: FIX ID collision when loading a WAL with multiple segments. #10587
```
Comment (roidelapluie) [inline CHANGELOG.md:5]:
```suggestion
* [BUGFIX] Agent: Fix ID collision when loading a WAL with multiple segments. #10587
```

**Comment by roidelapluie** [comment]
Thanks, two suggestions

**Review by roidelapluie — APPROVED** [review]

**Comment by roidelapluie** [comment]
I have restarted prombench. I do not expect us to cut the release on Monday anyway because it will be a day off for @Nexucis . However prombench should have been updated for go1.18

**Comment by Nexucis** [comment]
nice it looks like the memory leak is fixed. Well done @roidelapluie .
It also looks like using go 1.18 is reducing the usage of the memory. See the memory usage between this PR and the version v2.34

**Pull request merged**